components/expression/expression-panel.html

```diff
@@ -31,7 +43,7 @@
       </tr>
       <tr>
         <td>Condition Labels</td>
-        <td><input type="checkbox" id="label-conditions" title="show experiment condition (condition) labels" checked></td>
```

Comment:
It would lead to errors if we check the condition labels by default, because the view window is not large enough to display the whole labels.